_posts/2019-07-29-vim-basics.markdown

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: vim Basics
+date: 2019-07-29 22:28:02
+categories:
+author: bronzdoc
+---
+
+# VIM basics
+
+I would not bore you by telling you the history of vim, and the differences between vi and vim and all of its unix philosophy....
+If you are interested in that, you can give it a look [here](https://en.wikipedia.org/wiki/Vim_(text_editor).
```

Comment:
this link is not working (not correctly formatted), you probably need to remove a parentheses